src/lua/api-gateway/tracking/RequestTrackingManager.lua

```diff
@@ -30,7 +30,7 @@ local cjson = require "cjson"
 
 local _M = {}
 
-local KNWON_RULES =
+local KNOWN_RULES =
```

Comment:
ha, good catch @joshenders . Mind replacing in the other places where it's used ?

Reply:
Whoops. Added.